### ISSUE TITLE
Incompatible with stable ember-source/types

### ISSUE THREAD

**Issue by HeroicEric** [buschtoens/ember-link#784]
When using the [stable TypeScript Types in Ember 5.1](https://blog.emberjs.com/stable-typescript-types-in-ember-5-1) there is an issue with the types for ember-link:

```
node_modules/ember-link/dist/services/link-manager.d.ts:39:35 - error TS2315: Type 'PublicTransition' is not generic.

39     get currentTransitionStack(): Transition<unknown>[] | undefined;
```

**Comment by villander** [comment]
I think that's a related issue after bumping my codebase to `tsconfig/ember` 3.x I got a similar error

```Cannot find module 'ember-link/dist/services/link-manager' or its corresponding type declarations.```

**Comment by gossi** [comment]
It's released as [`3.2.0`](https://github.com/buschtoens/ember-link/releases/tag/v3.2.0)
